src/cloud/docker/docker-split-db.md

```diff
@@ -0,0 +1,187 @@
+    environment:
+      - MYSQL_ROOT_PASSWORD=magento2
+      - MYSQL_DATABASE=magento2
+      - MYSQL_USER=magento2
+      - MYSQL_PASSWORD=magento2
+    ports:
+      - '3306'
+    volumes:
+      - 'magento-db-sales:/var/lib/mysql'
+      - 'docker-entrypoint-sales:/docker-entrypoint-initdb.d'
+      - 'mariadb-conf:/etc/mysql/mariadb.conf.d'
+      - 'docker-mnt:/mnt:delegated'
+    networks:
+      magento:
+        aliases:
+          - db-sales.magento2.docker
+...
+```
+     
+6. The file `./.docker/config.php.dist` was generated with next configurations:
+
+```php
+<?php
+
+return [
+    'MAGENTO_CLOUD_RELATIONSHIPS' => base64_encode(json_encode([
+       //...
+        'database-sales' => [
+            [
+                'host' => 'db-sales',
+                'path' => 'magento2',
+                'password' => 'magento2',
+                'username' => 'magento2',
+                'port' => '3306'
+            ]
+        ],
+        'database-quote' => [
+            [
+                'host' => 'db-quote',
+                'path' => 'magento2',
+                'password' => 'magento2',
+                'username' => 'magento2',
+                'port' => '3306'
+            ]
+        ],
+    ])),
+   //...
+];
+```
+
+7. Run 
+
+```shell script
+docker-compose up -d
+```
+
+8. Run command 
+
+```shell script
+docker-compose run deploy bash
+```
+
+9. In the environment variable, 'MAGENTO_CLOUD_RELATIONSHIPS' of "deploy" service has also added data to access new database services. 
+Run command in the container: ` echo $MAGENTO_CLOUD_RELATIONSHIPS | base64 -d | json_pp `:
+
+```json
+{
+   "database-quote" : [
+      {
+         "host" : "db-quote",
+         "port" : "3306",
+         "path" : "magento2",
+         "username" : "magento2",
+         "password" : "magento2"
+      }
+   ],
+   "database-sales" : [
+      {
+         "path" : "magento2",
+         "port" : "3306",
+         "username" : "magento2",
+         "host" : "db-sales",
+         "password" : "magento2"
+      }
+   ]
+}
+ ```
+
+10. Yuo can use this credentials to connect to DB services
+11. Exit from container and run `docker-compose ps`:
+
+```shell script
+docker-compose ps
+            Name                           Command                  State                Ports
+-------------------------------------------------------------------------------------------------------
+...
+magento-cloud_db-quote_1        docker-entrypoint.sh mysqld      Up             0.0.0.0:32873->3306/tcp
+magento-cloud_db-sales_1        docker-entrypoint.sh mysqld      Up             0.0.0.0:32874->3306/tcp
+magento-cloud_db_1              docker-entrypoint.sh mysqld      Up             0.0.0.0:32872->3306/tcp
+...
+```
+
+12. Yuo can use this ports to connect to DB services from your host
+ 
+Now everything is ready to install Magento with Split Databases
+
+
+   ![Xdebug Helper options]({{ site.baseurl }}/common/images/cloud-xdebug_helper-options.png){:width="400px"}
```

Comment:
Not sure why this image is here.  Maybe you can remove it. Or if you need it for something, add a sentence to explain why it is here.

Reply:
Fixed